product_custom_attribute/models/attribute_attribute.py

```diff
@@ -66,7 +66,11 @@ def write(self, vals):
                             option_id = self.env['attribute.option'].browse(
                                 [option_change[1]])
                             if option_id.value_ref:
-                                product.write(
-                                    {custom_field: [(3, option_id.value_ref.id, 0)]})
+                                if self.attribute_type == 'select':
```

Comment:
@hparfr this might fix [the bug you mentioned here.](https://github.com/shopinvader/odoo-pim/pull/6#issuecomment-609770326)
I forgot to treat differently the many2one fields and the many2many, now it's ok.

Reply:
yes it looks like, will test it tomorrow